src/server/snapshot.h

```diff
@@ -92,18 +93,20 @@ class SliceSnapshot {
   void IterateBucketsFb(const Cancellation* cll, bool send_full_sync_cut);
 
   // Called on traversing cursor by IterateBucketsFb.
-  bool BucketSaveCb(PrimeIterator it);
+  bool BucketSaveCb(PrimeIterator it) ABSL_LOCKS_EXCLUDED(bucket_ser_mu_);
```

Comment:
@chakaz from my understanding `ABSL_LOCKS_EXCLUDED` roughly means that the `caller` of this function `should not hold the lock bucket_ser_mu_`. Isn't this redundant since `util::fb2::lock` actually checks for this `  70     CHECK(active != owner_);` ?